R/visualize.R

```diff
@@ -19,7 +19,7 @@
 #' @return A ggplot of MetAnnotate data
 #' @export
 generate_ggplot <- function(metannotate_data, hit_totals, plotting_colour_data,
-                            plotting_taxon, normalizing_HMM, plot_type = "bar",
+                            plotting_taxon, normalizing_HMM, plot_type = "bar", wrap = "vertical",
```

Comment:
@NickiShaw Please add a @param entry in the function descriptor above to describe the `wrap` variable